.github/workflows/pr-checks.yml

```diff
@@ -20,3 +20,12 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-python@v5
       - uses: pre-commit/action@v3.0.0
+  spell-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+      - name: Install cspell
+        run: npm install -g cspell
+      - name: Run Spell Checker
+        run: find ./content -name "*.md" | xargs cspell
```

Comment:
Could use this action instead, but this is fine aswell. I might replace it later with the action, unless you want to do that
https://github.com/streetsidesoftware/cspell-action

Reply:
That one is even better, update the code to use it now.

Reply:
@chrheg Seems like it has issues reading the config file
https://github.com/FidelusAleksander/ghcertified/actions/runs/7596864581/job/20691056083?pr=159

Reply:
Had to add the name of the config file to the action, looks like its running now.